Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/OTPAuthorizationResult.kt

```diff
@@ -1,19 +1,19 @@
 package de.rki.coronawarnapp.datadonation
 
 import androidx.annotation.Keep
-import com.google.gson.annotations.SerializedName
+import com.fasterxml.jackson.annotation.JsonProperty
 import java.time.Instant
 import java.util.UUID
 
 @Keep
 data class OTPAuthorizationResult(
```

Comment:
Please add the package to pro-guard to avoid release build issues, or did you check the release build ?
see how other models that are using Jackson are references in pro-guard file 
For more info see https://github.com/FasterXML/jackson-module-kotlin#caveats

Reply:
Removed here and added to proguard-rules.pro